docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -1019,6 +1019,16 @@ You can use the [`collapse-rename-collapsedheight` codemod](https://github.com/m
   + <IconButton size="large">
   ```
 
+- `span` element that wraps children has been removed. `label` classKey is also removed. Read more about [span removal](https://github.com/mui-org/material-ui/pull/26666).
```

Comment:
I think developer might be curious about why it is removed, so link to the description of the PR.